src/xrc-tests/src/mock_responses/stablecoin.rs

```diff
@@ -0,0 +1,76 @@
+use serde_json::json;
+use xrc::{Exchange, EXCHANGES};
+
+use crate::container::{ExchangeResponse, ResponseBody};
+
+fn sample_stablecoin_json(exchange: &Exchange) -> ResponseBody {
+    let json = match exchange {
+        Exchange::Binance(_) => json!([[
+            1614596340000i64,
+            "1.00",
+            "1.00",
+            "1.00",
+            "1.00",
+            "1.00",
```

Comment:
Would it make sense to introduce some variability here?

Reply:
Added some variability to the numbers. I may add more mocks to introduce more in order to trigger different error types as well.